stdlib/CMakeLists.txt

```diff
@@ -107,6 +107,31 @@ swift_create_stdlib_targets("swift-stdlib" "sibopt" TRUE)
 swift_create_stdlib_targets("swift-stdlib" "sibgen" TRUE)
 swift_create_stdlib_targets("swift-test-stdlib" "" FALSE)
 
+foreach(sdk ${SWIFT_SDKS})
+  foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES})
+    set(platform "${SWIFT_SDK_${sdk}_LIB_SUBDIR}")
+    set(input "${SWIFT_SOURCE_DIR}/stdlib/public/legacy_layouts/${platform}/layouts-${arch}.yaml")
+    set(output "${SWIFTLIB_DIR}/${platform}/layouts-${arch}.yaml")
```

Comment:
This'll copy them into the build directory, but I think you still need a `swift_install_in_components` for it to show up in a toolchain. (Test with "swift-ci build toolchain".)